ui/src/main/less/stageview_adjunct.less

```diff
@@ -8,6 +8,12 @@
 // Selectively import some twitter bootstrap styles.  Namespace them too.
 .cbwf-widget {
   @import "bootstrap";
+  
+  pre {
+    background-color: var(--background);
```

Comment:
Is this backwards-compatible?

Reply:
yes, as it's overriding bootstrap styles, it will fallback to the previous styles otherwise

Reply:
👍